bin/run_miniQMCPack.sh

```diff
@@ -0,0 +1,72 @@
+#! /usr/bin/env bash
+
+# run_miniQMCPack.sh - runs several MiniQMC binaries
+# The user should set at least these two variables:
+# MQMC_BUILD_PREFIX       miniqmc build prefix (default: ~/miniqmc_build)
```

Comment:
Move these env vars to the "user can set" section.  No env vars need to be set to run this script.